src/sentry/api/endpoints/project_transaction_names.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.exceptions import ParseError
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.utils import get_date_range_from_stats_period
+from sentry.ingest.transaction_clusterer.datasource import fetch_unique_transaction_names
+from sentry.ingest.transaction_clusterer.tree import TreeClusterer
+
+
+@region_silo_endpoint
+class ProjectTransactionNamesCluster(ProjectEndpoint):
```

Comment:
Should we add the endpoint to the code owners, even if it's a temporary one?

Reply:
done